packages/editor/src/components/rich-text/index.js

```diff
@@ -738,13 +743,48 @@ export class RichText extends Component {
 		}
 	}
 
+	/**
+	 * Applies given annotations to the given record.
+	 *
+	 * @param {Object} record The record to apply annotations to.
+	 * @param {Array} annotations The annotation to apply.
+	 * @return {Object} A record with the annotations applied.
+	 */
+	applyAnnotations( record, annotations = [] ) {
```

Comment:
Why is this a class method? It doesn't use `this` at all.

Reply:
Done.